data/starts.txt

```diff
@@ -0,0 +1,207 @@
+	`	The pen is heavy as lead but glides almost without friction as you print your initials on the bottom of page after page of ominous legal documents and sign several dozen statements, notices, and agreements. The banker, a balding middle-aged man in a suit that you suspect costs more than the spaceship you are about to purchase, flips the pages past you quickly, uttering a curt, "Sign here. Now here," as he points to each page. He moves swiftly, mechanically.`
+	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 433,567 credits in interest. Sign here." He grins.`
+	`	You sign your name one final time:`
+	name
+	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
+	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`
+
+
+
+start "hai space"
+	name "New Hai"
+	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending mostly locations in human space.`
+	description `	Now, as you enter the shipyard, you feel like this is the start of a new life as a captain.`
+	thumbnail "scene/hai start"
+	system "Fah Soom"
+	planet "Greenwater"
+	date 16 11 3013
+	to display
+		or
+			has "main plot completed"
+			has "Ask the Hai about the Unfettered"
+	on display
+		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
+		thumbnail ""
+		date "3020"
+		planet "Slightly too warm"
+		credits "over a million"
+	to reveal
+		has "main plot completed"
+		has "Ask the Hai about the Unfettered"
+	on reveal
+		name "Undiscovered Hai Start"
+		description "There could be interesting possibilities up north, once all crises are resolved."
+		description "	You could have started a different way, went a different path."
+		system "Somewhere far north"
+		planet "Mixed culture"
+		date "Same old one"
+		credits 1.431M
+		debt 1.431M
+	to unlock
+		has "main plot completed"
+		has "event: hai-human resolution announced"
```

Comment:
This won't be unlockable until HR is finished. Do we want to change this, or leave it until that campaign does get finished?

Reply:
```suggestion
		#has "event: hai-human resolution announced"
```
I'd just comment it out for now

Reply:
Could you also add a TODO to it, so we can come back to it easier after HR?